run/views.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework.decorators import api_view
+from rest_framework.parsers import JSONParser
+from .serializers import RunSerializer
+from .models import Run
+from rest_framework.response import Response
+
+
+@api_view(http_method_names=['POST'])
```

Comment:
The run::run method? That registers or in other words Create runs to be scheduled and run later. It returns the requested runs that are not valid in a list. 
If by "successful runs" you mean successfully registered, it does not return them. 
Completed runs can be queried via run::report method.